optaplanner-examples/src/main/java/org/optaplanner/examples/cheaptime/domain/CheapTimeSolution.java

```diff
@@ -16,36 +16,39 @@
 
 package org.optaplanner.examples.cheaptime.domain;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-
 import com.thoughtworks.xstream.annotations.XStreamAlias;
 import com.thoughtworks.xstream.annotations.XStreamConverter;
-import org.optaplanner.core.api.domain.solution.PlanningEntityCollectionProperty;
-import org.optaplanner.core.api.domain.solution.PlanningSolution;
-import org.optaplanner.core.api.domain.solution.Solution;
+import org.optaplanner.core.api.domain.solution.*;
 import org.optaplanner.core.api.domain.valuerange.ValueRangeProvider;
 import org.optaplanner.core.api.score.buildin.hardmediumsoftlong.HardMediumSoftLongScore;
 import org.optaplanner.core.impl.score.buildin.hardmediumsoftlong.HardMediumSoftLongScoreDefinition;
 import org.optaplanner.examples.common.domain.AbstractPersistable;
 import org.optaplanner.persistence.xstream.impl.score.XStreamScoreConverter;
 
+import java.util.List;
+
+// FIXME originally had itself as a fact
 @PlanningSolution
 @XStreamAlias("CtCheapTimeSolution")
-public class CheapTimeSolution extends AbstractPersistable implements Solution<HardMediumSoftLongScore> {
+public class CheapTimeSolution extends AbstractPersistable {
 
     private int timeResolutionInMinutes;
     private int globalPeriodRangeFrom; // Inclusive
     private int globalPeriodRangeTo; // Exclusive
 
+    @PlanningFactCollectionProperty
     private List<Resource> resourceList;
```

Comment:
Consistency: Each example should either use ALL **field annotations** or ALL **method annotations **(usually getter annotations).
Most examples currently use method annotations, including the CheapTimeSolution. This is because JPA-hibernate also recommends method annotations over field annotations (but I recently started prefering reading field annotations). In any case, we should mix in 1 example.

The taskassigning examples uses field annotations for @PlanningVariable etc, so it should use field annotations for @PlanningFactCollectionProperty too. But for all other examples, its' the other way around (unless we decide to switch a number of - not all of them - to field annotations, but I 'd rather decide that in isolation from this issue again).